### DISCUSSION TITLE
How to Add nvcuda.dll

### DISCUSSION THREAD

**Discussion by metedeh** [mpvnet-player/mpv.net#717] [Q&A]
How to add nvcuda.dll to mpvnet and fix this error? 

I use AMD Ryzen 5 5500U, Windows 10 64bit, RAM 8GB

![Screenshot 2024-11-27 080459](https://github.com/user-attachments/assets/e56a2e14-aab6-4886-822c-19649a2f223b)

**Comment by stax76** [comment]
I don't know what's wrong. You can request help for it on the mpv issue tracker. As title, you can use something like:

How can I solve this decoding error
